test/dotnet.Tests/CommandTests/Run/RunFileTests.cs

```diff
@@ -118,6 +118,33 @@ public void FilePath(string? path, bool differentCasing)
         }
     }
 
+    /// <summary>
+    /// <c>dotnet file.cs</c> is equivalent to <c>dotnet run file.cs</c>.
+    /// </summary>
+    [Fact]
+    public void FilePath_WithoutRun()
+    {
+        var testInstance = _testAssetsManager.CreateTestDirectory();
+        File.WriteAllText(Path.Join(testInstance.Path, "Program.cs"), s_program);
```

Comment:
It would be good to also try a version of `s_program` which prepends `#:property Configuration=Debug`